projects/goipp/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
```

Comment:
Same observation about the copyright ✌️

Reply:
[This is indeed expected](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#copyright-headers).